src/MicrosoftTeams.ts

```diff
@@ -455,6 +457,16 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * Retrieves client type of the host.
+   * Possible values are : desktop, web, android, ios
+   */
+  export function getHostClientType(): string {
```

Comment:
I think Vinay and I discussed adding this property to the context. Any particular reason you chose to add a dedicated getter for this? What happens if the initialize response hasn’t come back yet when this method is called? How does a dev know when it’s OK to call the API?